aat/src/aat/resources/features/F-094 NoC ChallengeQuestions/F-094.feature

```diff
@@ -0,0 +1,38 @@
+@F-094
+Feature: F-094 Retrieve ChallengeQuestions for Notice of Change
+
+  Background:
+    Given an appropriate test context as detailed in the test data source,
+    And a user [MCA System - with an IDAM role of caseworker CAA],
+
+  @S-094.1
+  Scenario: Must successfully return the contents of the ChallengeQuestion definition
+    Given the request [intends to get the contents of a set of questions],
+    And the request [contains a valid CaseType ID],
+    And the request [contains a valid Collection Question ID],
+    And a request is prepared with appropriate values,
+    When it is submitted to call the [GET ChallengeQuestion] operation of [CCD Definition Store Api],
```

Comment:
The operation name should be consistent with the name fo the feature, as this feature is itself the operation under test.